os/ProtectedLibs.mk

```diff
@@ -110,6 +110,7 @@ endif
 
 # Add libraries for compression module
 
+#ifeq ($(CONFIG_COMPRESSION),y)
 ifeq ($(CONFIG_COMPRESSED_BINARY),y)
```

Comment:
How about having CONFIG_COMPRESSION as the parent configuration and CONFIG_COMPRESSED_BINARY as an option under the COMPRESSION menu?

Reply:
config_compression is to select compression method. we add log dump, app compression dependency to it.
We will restructure config_compressed_binary with rename etc in new PR